src/app/dynamic-forms/cinchy-dynamic-forms.component.ts

```diff
@@ -408,6 +422,20 @@ export class CinchyDynamicFormsComponent implements OnInit, OnChanges {
     });
   }
 
+  createNewRecord(): void {
```

Comment:
There should be two blank lines before functions and one blank line directly after the function header (just for consistency)